cookiecutter.json

```diff
@@ -1,21 +1,14 @@
 {
-    "client_name": "ds",
-    "project_name": "default",
-    "__project_name_slug": "{{cookiecutter.project_name | slugify}}",
-    "__client_name_slug": "{{cookiecutter.client_name | slugify}}",
-    "repo_name": "{{cookiecutter.__client_name_slug}}-{{cookiecutter.__project_name_slug}}",
-    "ci": [
-      "GitLab",
-      "Github",
-      "None"
-    ],
-    "jupytext": [
-      "No",
-      "Yes"
-    ],
-    "python_package_name": "{{ cookiecutter.__project_name_slug.replace('-', '_') }}",
-    "__package_name": "{{ cookiecutter.python_package_name }}",
-    "_copy_without_render": [
-      ".github/workflows/*.yml"
-    ]
-  }
\ No newline at end of file
+  "client_name": "ds",
+  "project_name": "default",
+  "__project_name_slug": "{{cookiecutter.project_name | slugify}}",
+  "__client_name_slug": "{{cookiecutter.client_name | slugify}}",
+  "repo_name": "{{cookiecutter.__client_name_slug}}-{{cookiecutter.__project_name_slug}}",
+  "ci": ["Github", "GitLab", "None"],
+  "jupytext": ["No", "Yes"],
+  "docs": ["Sphinx", "No docs"],
+  "versioning": ["Python Semantic Release", "Bumpversion", "None"],
```

Comment:
do we support python semantic already? :thinking:

Reply:
Tak, ale tylko dla GitHub CI/CD